src/components/procedure/_add-demo-project.mdx

```diff
@@ -0,0 +1,13 @@
+:::info Prerequisite
```

Comment:
FYI - The "Explore a demo project" button appears by default for any new user of Semgrep because they have no projects. They don't have to do anything. For older accounts like yours and mine, it doesn't appear "by default".

Reply:
@s-santillan yes, I had to create a new organization without any projects to test this. Is the prerequisite technically correct or does this require any edits?

Reply:
Should I add something as: If the ***Explore a demo project*** button is not displayed on your Projects page, you may already have other projects in your Semgrep App organization.

Reply:
@s-santillan let me know if this looks good. https://github.com/returntocorp/semgrep-docs/pull/820#discussion_r1047660809